Refactor the ValueSet, BatchRequest, and PlugConfig types and routes

yash-learner opened the pull request:
## Proposed Changes

Fixes sub task in  https://github.com/ohcnetwork/care_fe/issues/12909
This pull request refactors and standardizes the ValueSet and PlugConfig APIs and related components across the codebase. It removes legacy `PlugConfig` and batch request routes and components, migrates all ValueSet-related code to use the new `valueSetApi` and updated types, and improves consistency in naming and usage. The changes also update components and hooks to use the new ValueSet types and API functions, ensuring better maintainability and clarity.

### API and Route Cleanup

* Removed all `PlugConfig`-related routes and components from `AppRouter.tsx` and `api.tsx`, including `/apps`, `/apps/plug-configs/:slug`, and associated API endpoints. Also removed batch request endpoints. [[1]](diffhunk://#diff-ba7691a84a8d897867e49026ab18bfd9b8899feb6dd37388d2c5e7989fcbf7a5L27-L28) [[2]](diffhunk://#diff-ba7691a84a8d897867e49026ab18bfd9b8899feb6dd37388d2c5e7989fcbf7a5L78-L80) [[3]](diffhunk://#diff-ce6faa5d5e7bcf19601a5ec053a7a08e5a59a5193d0c65dc1ae383f460526745L2-L5) [[4]](diffhunk://#diff-ce6faa5d5e7bcf19601a5ec053a7a08e5a59a5193d0c65dc1ae383f460526745L49-L97)

### ValueSet API Migration

* Migrated all ValueSet-related API calls and types from legacy `valuesetApi` and types in `@/types/valueset/valueset` to new `valueSetApi` and types in `@/types/valueSet/valueSet` across all components, including `CodingEditor`, `SelectOrCreateValueset`, `ValueSetSearchContent`, `CodingField`, and `ValueSetEditor`. [[1]](diffhunk://#diff-243c6c4d934e22722e7b33ca529e36e139cb306d7128c6d5660fdc6be22da5bdL28-R34) [[2]](diffhunk://#diff-243c6c4d934e22722e7b33ca529e36e139cb306d7128c6d5660fdc6be22da5bdL50-R51) [[3]](diffhunk://#diff-7b2e51950b8553d083cacaabb24ee3da6d454435e1b73a24c3adea13fa261462R13-L17) [[4]](diffhunk://#diff-7b2e51950b8553d083cacaabb24ee3da6d454435e1b73a24c3adea13fa261462L29-R29) [[5]](diffhunk://#diff-7b2e51950b8553d083cacaabb24ee3da6d454435e1b73a24c3adea13fa261462L38-R49) [[6]](diffhunk://#diff-01f0f3ef303c87d2703792b999683df81a7bdf1a72567760b5355e021046e22cL21-R24) [[7]](diffhunk://#diff-af1f62b77c5cdef4ab267fe2feb8905fecda501cdf4bfd2fd5f9d2a3e7b0842fR17-L19) [[8]](diffhunk://#diff-af1f62b77c5cdef4ab267fe2feb8905fecda501cdf4bfd2fd5f9d2a3e7b0842fL41-R42) [[9]](diffhunk://#diff-7aa126245d8e71d65fc1ee29aaab10966624fe57235dc6bdf8c899dcc2f7dde5R8-R22) [[10]](diffhunk://#diff-7aa126245d8e71d65fc1ee29aaab10966624fe57235dc6bdf8c899dcc2f7dde5L31-R40) [[11]](diffhunk://#diff-7aa126245d8e71d65fc1ee29aaab10966624fe57235dc6bdf8c899dcc2f7dde5L49-R68)

### Type and Naming Standardization

* Updated types and props in ValueSet-related components to use new `ValueSetBase`, `ValueSetRead`, `ValueSetCreate`, `ValueSetUpdate`, and `ValueSetStatus` types, replacing old types like `ValuesetBase`, `ValuesetLookupResponse`, and `ValuesetFormType`. [[1]](diffhunk://#diff-46d472967b3635efce0e043ca35db1fbb397d8e0bdfed8a16d60907884c6c6d1L33-R45) [[2]](diffhunk://#diff-46d472967b3635efce0e043ca35db1fbb397d8e0bdfed8a16d60907884c6c6d1L57-R58) [[3]](diffhunk://#diff-46d472967b3635efce0e043ca35db1fbb397d8e0bdfed8a16d60907884c6c6d1L113-R114) [[4]](diffhunk://#diff-7aa126245d8e71d65fc1ee29aaab10966624fe57235dc6bdf8c899dcc2f7dde5R8-R22) [[5]](diffhunk://#diff-7aa126245d8e71d65fc1ee29aaab10966624fe57235dc6bdf8c899dcc2f7dde5L49-R68)


### Minor UI and Behavior Improvements

* Improved translation support for labels in `CodingEditor`, fixed icon rendering in `ValueSetSearchContent`, and ensured correct mutation payloads for clearing favourites. [[1]](diffhunk://#diff-243c6c4d934e22722e7b33ca529e36e139cb306d7128c6d5660fdc6be22da5bdL96-R96) [[2]](diffhunk://#diff-01f0f3ef303c87d2703792b999683df81a7bdf1a72567760b5355e021046e22cL74-R73) [[3]](diffhunk://#diff-01f0f3ef303c87d2703792b999683df81a7bdf1a72567760b5355e021046e22cL390-R393)
- Change 1
- Change 2
- Additional context if needed

Tagging: @ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate the bug or test the new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is placed in I18n files.
- [ ] Prepare a screenshot or demo video for the changelog entry and attach it to the issue.
- [ ] Request peer reviews.
- [ ] Complete QA on mobile devices.
- [ ] Complete QA on desktop devices.


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **Refactor**
  * Removed plugin configuration management functionality and associated routes from the application.
  * Refactored value set system with consolidated type structures and updated API surface.
  * Migrated value set status handling to use an enumerated type with integrated UI icon and color mappings.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

yash-learner committed:
WIP: Move file api routes and its types

yash-learner committed:
WIP: Move file api routes and its types

yash-learner committed:
Some tiny fixes -> during self review so far

yash-learner committed:
Some tiny fixes -> during self review so far

yash-learner committed:
Clean up the markUploadComplete and create mutation calls as per current expected BE api

yash-learner committed:
Move file related constants to file.ts adn some clean up

yash-learner committed:
Move file related constants to file.ts adn some clean up

yash-learner committed:
Elaborate path param name

yash-learner committed:
Move usre related routes and update the useAvatar to use useMutation

yash-learner committed:
Replace UserBearMinimal with new UserReadMinimal

yash-learner committed:
Clean UseFacilityMOdel and AuthUser type and use new one's

yash-learner committed:
Remove unused file

yash-learner committed:
Tiny redundant fix

yash-learner committed:
Remove redundant translation

yash-learner committed:
Let's keep things as rithvik suggested, let keep it broke until BE fixes the url

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr3

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr3

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr3

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr3

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr3' into issues/12909/migrate-from-cr-osr-pr4

yash-learner committed:
Update types based on latest backend

yash-learner committed:
Refactor auth related routes and types

yash-learner committed:
Move MFA related types from otp to auth

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr3

yash-learner committed:
Fix lint

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr3

yash-learner committed:
Refactor: migrate OTP types and routes to new structure, replacing PatientRead with PublicPatientRead across components and APIs

yash-learner committed:
Invalidate with the associating_id

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr3

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr3' into issues/12909/migrate-from-cr-osr-pr4

yash-learner committed:
Fix a tiny repetition

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr4' into issues/12909/migrate-from-cr-osr-pr6-authapis

yash-learner committed:
Fix the naming case used

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr6-authapis' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Fix eslint error

yash-learner committed:
Refactor types and routes

Refactor resource request handling: update API routes,
replace constants with enums,
and enhance localization keys for resource statuses and categories.

yash-learner committed:
Refactor resource request comment api routes and types

yash-learner committed:
Add as const

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr7-otpapis' into issues/12909/migrate-from-cr-osr-pr8-rrc-apis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr6-authapis

yash-learner committed:
wip: type and route refactor for batch and valuset

yash-learner committed:
This should wrap the valueset api route and type cleanup

- other minor things

yash-learner committed:
Clean up deprecated plug related routes, types and components

AdityaJ2305 commented on src/components/Questionnaire/ValueSetSearchContent.tsx:
```diff
@@ -387,7 +390,7 @@ export default function ValueSetSearchContent({
         disabled={clearFavouritesMutation.isPending}
         onConfirm={() => {
-          clearFavouritesMutation.mutate({});
+          clearFavouritesMutation.mutate(undefined);
```
```suggestion
          clearFavouritesMutation.mutate();
```

ps: I'm sorry I know it's in work in progress state but this thing just caught my eyes

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr6-authapis' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Remove unnecessary params

yash-learner committed:
Some tiny fixes in auth

-  Add a type for token refresh
-  Update some things

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Remove undefined

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Fix the order

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Move age type conversion from onchnage to onsubmit

yash-learner committed:
Change age field type from number in schema

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr7-otpapis' into issues/12909/migrate-from-cr-osr-pr8-rrc-apis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr7-otpapis' into issues/12909/migrate-from-cr-osr-pr8-rrc-apis

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr8-rrc-apis' into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr7-otpapis' into issues/12909/migrate-from-cr-osr-pr8-rrc-apis

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr8-rrc-apis' into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr7-otpapis

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr7-otpapis' into issues/12909/migrate-from-cr-osr-pr8-rrc-apis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr8-rrc-apis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr8-rrc-apis

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr8-rrc-apis

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr8-rrc-apis' of https://github.com/ohcnetwork/care_fe into issues/12909/migrate-from-cr-osr-pr8-rrc-apis

yash-learner committed:
Fix assigned_facility type

yash-learner committed:
Keep null handling similar

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr8-rrc-apis

yash-learner committed:
Remove unused ResourceRequestUpdate interface

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr8-rrc-apis

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr8-rrc-apis' into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner committed:
Update the endpoint name

yash-learner committed:
Update the api route

yash-learner committed:
Fix typo

yash-learner commented on src/pages/Facility/settings/observationDefinition/ObservationInterpretation.tsx:
```diff
@@ -994,7 +994,7 @@ function CustomValueSetInterpretationComponent({
 
   const { data: valuesets } = useQuery({
     queryKey: ["valusets"],
-    queryFn: query(valuesetApi.list),
+    queryFn: query(valueSetApi.list),
   });
```
https://github.com/ohcnetwork/care_fe/pull/13568/commits/c9b9fcfd015614c066517dd974e2b58409d9329a

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr9-others

gigincg commented on src/types/base/code/code.ts:
```diff
@@ -1,9 +1,17 @@
 import { z } from "zod";
+export interface Designation {
+  language: string;
+  use: Code;
+  value: string;
+}
```
@yash-learner I think we should try and resolve this

yash-learner replied:
@gigincg @coderabbitai

Resolve it here https://github.com/ohcnetwork/care_fe/pull/13568/commits/948b1cb47e87bed547f11dc25c99385e670a1c71, added proper separate type called `CodeConceptMinimal` based on BE https://github.com/ohcnetwork/care/blob/b90023707100578cc45d102e693cca720f0a775a/care/emr/fhir/resources/code_concept.py#L24

gigincg commented on src/components/Questionnaire/ValueSetSearchContent.tsx:
```diff
@@ -98,19 +97,23 @@ export default function ValueSetSearchContent({
+      body: {
+        count,
+        search: search + searchPostFix,
+        language: DEFAULT_EXPAND_REQUEST_LANGUAGE,
```
Does the backend support this? 

If so, I think DEFAULT_EXPAND_REQUEST_LANGUAGE should be overridable via env

yash-learner replied:
> Does the backend support this?

I think no it is currently set to default as `en-gb` and i tried changing the languages to `es` and `zn-ch` it did not filter out

gigincg replied:
If the backend doesn't support it, let's please not have it.

yash-learner replied:
Yep, fixed it https://github.com/ohcnetwork/care_fe/pull/13568/commits/8f775484e04d7c31acc2ff6e60a1b3f10bf246db here

gigincg replied:
Not removing it here? And the DEFAULT_EXPAND_REQUEST_LANGUAGE seems void of any purpose as well

yash-learner replied:
@gigincg 

I am confused 

I removed it in the commit I mentioned  above, https://github.com/ohcnetwork/care_fe/commit/8f775484e04d7c31acc2ff6e60a1b3f10bf246db 

The file on the PR branch  https://github.com/ohcnetwork/care_fe/blob/issues/12909/migrate-from-cr-osr-pr9-others/src/components/Questionnaire/ValueSetSearchContent.tsx#L104 and in the commit  it does not show that line

Looks like GitHub issue, cache issue or github bug

Can you please check the diff of my commit 

<img width="3589" height="1613" alt="Image" src="https://github.com/user-attachments/assets/f8698fe7-650d-48b5-bba1-b8d81caf535e" />

yash-learner replied:
Can you please check using `changes from all commits` https://github.com/ohcnetwork/care_fe/pull/13568/files#diff-01f0f3ef303c87d2703792b999683df81a7bdf1a72567760b5355e021046e22c  this should show updated changes

gigincg commented on src/types/base/batch/batchApi.ts:
```diff
@@ -4,7 +4,7 @@ import { BatchRequestResponse } from "@/types/base/batch/batch";
 
 const batchApi = {
-  batchRequest: {
+  request: {
```
I feel it would be better to keep it as batchRequest, since we're so used to the term

yash-learner replied:
Reverted it https://github.com/ohcnetwork/care_fe/pull/13568/commits/085cf517d74325811978daaadd565016516ad7d9

gigincg commented on src/types/valueSet/valueSet.ts:
```diff
@@ -0,0 +1,138 @@
+  sufficientlyDefined: boolean | null;
+}
+
+export interface DesignationItem {
```
Can you confirm that these types would work with all FHIR valuesets? Not just SNOMED valuesets

yash-learner replied:
Addressed it here https://github.com/ohcnetwork/care_fe/pull/13568/commits/948b1cb47e87bed547f11dc25c99385e670a1c71#diff-1d5198e1b5bcb5eed1b7117d0954010f3be22f3e3973396dec54b52a49c9d30e

yash-learner committed:
Keep it as batchRequest

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner committed:
Address some review comments

- Add type with designation based on BE - CodeConceptMinimal
- Fix the property and designations type

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr9-others' of https://github.com/ohcnetwork/care_fe into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner commented on src/types/valueSet/valueSetApi.ts:
```diff
@@ -1,102 +1,102 @@
-    TBody: Type<ValuesetLookupRequest>(),
+    TBody: Type<ValueSetLookupRequest>(),
+    TRes: Type<ValueSetLookupResponse>(),
   },
```
that is typo in old code and it was using the correct  endpoint from `api.tsx`

yash-learner committed:
MOve DEFAULT_EXPAND_REQUEST_LANGUAGE constant to valueSet.ts

yash-learner committed:
Fix a thing from this commit https://github.com/ohcnetwork/care_fe/pull/13568/commits/948b1cb47e87bed547f11dc25c99385e670a1c71#diff-01f0f3ef303c87d2703792b999683df81a7bdf1a72567760b5355e021046e22c

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner commented on src/hooks/useLocationManagement.ts:
```diff
@@ -88,7 +88,7 @@ export function useLocationManagement({
       );
 
-      return mutate(routes.batchRequest, { silent: true })({
+      return mutate(batchApi.batchRequest, { silent: true })({
```
I reverted it based on review comment  https://github.com/ohcnetwork/care_fe/pull/13568#discussion_r2437092880

nihal467 committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr9-others

gigincg commented on src/components/Questionnaire/ValueSetSearchContent.tsx:
```diff
@@ -394,7 +397,7 @@ export default function ValueSetSearchContent({
         disabled={clearFavouritesMutation.isPending}
         onConfirm={() => {
-          clearFavouritesMutation.mutate({});
+          clearFavouritesMutation.mutate();
```
This seems relevant

yash-learner replied:
@gigincg 

I think choosing any one is fine here because even without explicit `undefined` the body will be empty because how typescript handle `void` parameter which allows omitting it -> which would at runtime be undefined

`For void parameters: we can call the function with NO arguments OR with undefined`

**with clearFavouritesMutation.mutate()**

```
curl 'https://careapi.ohc.network/api/v1/valueset/system-condition-code/clear_favourites/' \
  -X 'POST' \
  -H 'accept: application/json' \
  -H 'accept-language: en-US,en;q=0.9' \
  -H 'authorization: Bearer xxx' \
  -H 'content-length: 0' \
  -H 'content-type: application/json' \
  -H 'origin: http://localhost:4000' \
  -H 'priority: u=1, i' \
  -H 'referer: http://localhost:4000/' \
  -H 'sec-ch-ua: "Google Chrome";v="141", "Not?A_Brand";v="8", "Chromium";v="141"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "Windows"' \
  -H 'sec-fetch-dest: empty' \
  -H 'sec-fetch-mode: cors' \
  -H 'sec-fetch-site: cross-site' \
  -H 'user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/141.0.0.0 Safari/537.36'

gigincg committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner commented on src/hooks/useLocationManagement.ts:
```diff
@@ -88,7 +88,7 @@ export function useLocationManagement({
       );
 
-      return mutate(routes.batchRequest, { silent: true })({
+      return mutate(batchApi.batchRequest, { silent: true })({
```
Updated the PR description  

I reverted it based on a previous review comment.

yash-learner committed:
Remove DEFAULT_EXPAND_REQUEST_LANGUAGE constant and its usage from ValueSetSearchContent component

yash-learner committed:
Merge branch 'issues/12909/migrate-from-cr-osr-pr9-others' of https://github.com/ohcnetwork/care_fe into issues/12909/migrate-from-cr-osr-pr9-others

yash-learner committed:
Address the copilot comment: Remove comment from ValueSetForm component

yash-learner commented on src/components/ValueSet/ValueSetForm.tsx:
```diff
@@ -31,19 +31,19 @@ import useAppHistory from "@/hooks/useAppHistory";
 
-// Create a schema for form validation
-
 interface ValueSetFormProps {
```
Addressed https://github.com/ohcnetwork/care_fe/pull/13568/commits/aba864b9fcf393c26545780e32b92f0a4f809b7f

yash-learner commented on src/components/ValueSet/ValueSetForm.tsx:
```diff
@@ -31,19 +31,20 @@ import useAppHistory from "@/hooks/useAppHistory";
 import { CodingField } from "./CodingField";
 import { ValueSetPreview } from "./ValueSetPreview";
 
 // Create a schema for form validation
```
Addressed https://github.com/ohcnetwork/care_fe/pull/13568/commits/aba864b9fcf393c26545780e32b92f0a4f809b7f

yash-learner committed:
Merge branch 'develop' into issues/12909/migrate-from-cr-osr-pr9-others
